app/helpers/get-section-id.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function getSectionId(params/*, hash*/) {
```

Comment:
Add docstring here to clarify what this helper is doing/what it's being used for